### ISSUE TITLE
White background shows when keyboard show on iOS9

### ISSUE THREAD

**Issue by codecoo** [driftyco/ionic-plugin-keyboard#159]
I have the keyboard problem build ionic on ios9.x
in app.js           
cordova.plugins.Keyboard.hideKeyboardAccessoryBar(false);
cordova.plugins.Keyboard.disableScroll(true);

plugin list & version

ionic-plugin-keyboard 1.0.8 "Keyboard"
  
Cordova CLI: 5.4.1
Ionic Version: 1.2.1
Ionic CLI Version: 1.7.12
Ionic App Lib Version: 0.6.5
ios-deploy version: 1.8.3 
ios-sim version: 4.1.1 
OS: Mac OS X El Capitan
Node Version: v0.12.7
Xcode version: Xcode 7.2 Build version 7C68 

video link  : https://www.youtube.com/watch?v=kekelBdVhDs

**Comment by keenan35i** [comment]
+1

**Comment by chompunut** [comment]
+1

**Comment by kndol** [comment]
+1

**Comment by IanRamosC** [comment]
I'm facing the same issue. Did you find a way to solve it?

**Comment by hare2011** [comment]
+1

**Comment by hare2011** [comment]
need help

**Comment by kenmickles** [comment]
This looks like a duplicate of https://github.com/driftyco/ionic/issues/4849. Updating  to Ionic 1.2.4 fixed the issue for me.

**Comment by lyd600lty** [comment]
@keenan35i Yes, 1.2.4 seems like fixing this problem, but when the input is at the bottom of screen, the page will be shaking once heavily.

**Comment by Bccorb** [comment]
Yes, this is a known bug with version 1.2.1. Going to version 1.2.3 fixed the issues and all subsequent versions as well. Also I believe going down to 1.2 also fixes the issues. I spent DAYS with this problem before I upgraded last year!

**Comment by VinceOPS** [comment]
+1 and yes, it looks like https://github.com/driftyco/ionic/issues/4849, however, issue still happening here (even with ionic 1.3).
iPhone 6, iOs 9.2, scrollable content (ion-content) with scroll required (that's why I can't use all the patchworks proposed by the community).
